File: tooling/sdkgen/test/typescript/openws-sdkgen-typescript-node/src/core/models/client/joined-room-payload.ts
export interface JoinedRoomPayloadInit {
    joinerId: string
    roomId: string
}

export class JoinedRoomPayload implements JoinedRoomPayloadInit {
    readonly joinerId: string
    readonly roomId: string

    constructor({ joinerId, roomId }: JoinedRoomPayloadInit = {} as JoinedRoomPayloadInit) {
        this.joinerId = joinerId
        this.roomId = roomId
    }
}

File: tooling/sdkgen/test/dotnet/UnityTestbed/Assets/chat/core/Runtime/Example.Chat.Core.Sdk/Models/Client/JoinedRoomPayload.cs
using Newtonsoft.Json;
using System.Collections.Generic;

namespace Example.Chat.Core.Models.Client
{
    public partial class JoinedRoomPayload
    {
        [JsonProperty("joinerId")]
        public string JoinerId;

        [JsonProperty("roomId")]
        public string RoomId;

    }
}